src/main/java/net/wurstclient/hacks/BoatFlyHack.java

```diff
@@ -45,7 +55,9 @@ public void onUpdate()
 		// fly
 		Entity vehicle = MC.player.getVehicle();
 		Vec3d velocity = vehicle.getVelocity();
-		double motionY = MC.options.keyJump.isPressed() ? 0.3 : 0;
-		vehicle.setVelocity(new Vec3d(velocity.x, motionY, velocity.z));

```

Comment:
I think over here, the space bar should be accounted for, and maybe increase the `motionY` value by 0.3, or some other configurable value.